src/lib.rs

```diff
@@ -54,7 +54,7 @@
 //! to the network through any node, and exchange public keys with it. That node becomes a
 //! bootstrap node for the client, and messages to and from the client will be routed over it.
 //!
-//! ```no_run
+//! ```ignore
```

Comment:
Ah, right! Not sure what's the best solution, but at some point it would be good to make this `no-run` again somehow, to make sure it stays up to date with the `Client` API.